src/clr-addons/progress-spinner/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+export * from './progress-spinner.module';
```

Comment:
I'm not sure if we also need to export the directive file here, as we never use the class itself directly.
For now let's try without it and see what happens in the web demo.